### PR TITLE
Docs: Add ZIP routes to non-JS integrations

### PR DESCRIPTION
**Pull request by gorzelinski**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: Add ZIP routes to non-JS integrations.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: docs/getting-started/integrations/dotnet.md [hunk 1]
```diff
@@ -20,29 +21,139 @@ You get ready-to-use code tailored to your needs!
 
 ## Setting up the project
 
-For the purpose of this guide, we will use a basic ASP.NET Core project created with `dotnet new webapp`. You can refer to the [ASP.NET Core documentation](https://learn.microsoft.com/en-us/aspnet/core/getting-started/?view=aspnetcore-7.0) to learn how to set up a project in the framework.
+For the purpose of these guides, we will use a basic ASP.NET Core project created with `dotnet new webapp`. You can refer to the [ASP.NET Core documentation](https://learn.microsoft.com/en-us/aspnet/core/getting-started/?view=aspnetcore-7.0) to learn how to set up a project in the framework.
```
Comment (godai78) [inline docs/getting-started/integrations/dotnet.md:24]: Why plural?

**Review by godai78 — APPROVED** [review]

File: docs/getting-started/integrations/dotnet.md [hunk 2]
```diff
@@ -115,8 +118,214 @@ Then, modify the `Index.cshtml` file in the `Pages` directory to include the CKE
...
+│   └── robots.txt
+├── resources
+│   ├── views
+|      ├── welcome.blade.php
+|      └── ...
+├── routes
+└── ...
+```
+
+Having all the dependencies of CKEditor&nbsp;5, modify the `welcome.blade.php` file in the `resources/views` directory to import them. All the necessary markup is in the HTML file from the ZIP archive &ndash; `index.html`. You can copy and paste it into your template. Pay attention to the paths of the import map and CSS link &ndash; they should reflect your folder structure. The template should look similar to the one below:
```
Comment (filipsobol) [inline docs/getting-started/integrations/dotnet.md:159]:
```suggestion
Having all the dependencies of CKEditor&nbsp;5, modify the `welcome.blade.php` file in the `resources/views` directory to import them. All the necessary markup is in the `index.html` file from the ZIP archive. You can copy and paste it into your template. Pay attention to the paths in the import map and CSS link &ndash; they should reflect your folder structure. The template should look similar to the one below:
```

Suggestion

File: docs/getting-started/integrations/laravel.md [hunk 3]
```diff
@@ -115,12 +116,113 @@ Then, modify the `welcome.blade.php` file in the `resources/views` directory to
...
+│   └── robots.txt
+├── resources
+│   ├── views
+|      ├── welcome.blade.php
+|      └── ...
+├── routes
+└── ...
+```
+
+Having all the dependencies of CKEditor&nbsp;5, modify the `welcome.blade.php` file in the `resources/views` directory to import them. All the necessary markup is in the HTML file from the ZIP archive &ndash; `index.html`. You can copy and paste it into your template. Pay attention to the paths of the import map and CSS link &ndash; they should reflect your folder structure. The template should look similar to the one below:
```
Comment (filipsobol) [inline docs/getting-started/integrations/laravel.md:159]:
```suggestion
Having all the dependencies of CKEditor&nbsp;5, modify the `welcome.blade.php` file in the `resources/views` directory to import them. All the necessary markup is in the `index.html` file from the ZIP archive. You can copy and paste it into your template. Pay attention to the paths in the import map and CSS link &ndash; they should reflect your folder structure. The template should look similar to the one below:
```

Same suggestion as above.

**Review by filipsobol — APPROVED** [review]

File: docs/getting-started/integrations/dotnet.md (hunk 1 above)
Comment (gorzelinski) [reply docs/getting-started/integrations/dotnet.md:24]: My thought process was: "This section is relevant to both paths - CDN and ZIP," so I used the plural. But, as I'm thinking now, I could also treat the whole guide as one. I don't know which is better 🤔

Comment (godai78) [reply docs/getting-started/integrations/dotnet.md:24]: I suggest treating the whole guide as one, it's just a guide for laravel or dotnet and that's it. Paths are something different in the nomenclature we use, and guides are documents.

**Pull request merged**